src/frontend/metrics/AppDiagnosticMetrics.ts

```diff
@@ -202,3 +210,27 @@ export class AppDiagnosticMetrics {
     }
   }
 }
+
+function getSelectedLanguageTag() {
+  const stored = storage.getString(LOCALE_STORAGE_KEY);
+
+  if (!stored) {
+    // TODO: Log to Sentry?
+    return null;
+  }
+
+  const parsed = JSON.parse(stored) as unknown;
+
+  if (
+    typeof parsed === 'object' &&
+    parsed !== null &&
+    'languageTag' in parsed &&
+    typeof parsed.languageTag === 'string'
+  ) {
+    return parsed.languageTag;
+  }
+
+  // TODO: Log to Sentry?
```

Comment:
wondering if we should log to sentry here because it means that the stored value does not have the expected field we want to extract